<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qianyueDialog</class>
 <widget class="QDialog" name="qianyueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/images/qianyuechengjiao.png</normaloff>:/images/qianyuechengjiao.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>房源业主</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>客源业主</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>中间人</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>佣金</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>分成</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBoxFang"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxKe"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxRen"/>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBoxYongjin"/>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="comboBoxFen"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加</string>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>删除</string>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>编号</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="spinBoxBianhao"/>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTableView" name="tableViewqianyue"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>应用编辑</string>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/images/modify.png</normaloff>:/images/modify.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
